plugins/org.jboss.tools.jmx.remoting1_1_3/.classpath

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
rather weird plugin name? not sure if we really want to include the version in the plugin name since we'd have to update the plugin name if we have to upgrade the jmx library rthat's included.